### PR TITLE
[NKS-1035] Custom SSH keys

### PR DESCRIPTION
**Pull request by sverrirsig**
Adds an array of ssh keys to the cluster provider config. This array is then injected into the cloud-init portion of the injection script.

### CODE REVIEW

**Comment by sverrirsig** [comment]
@ntfrnzn I'll have to update this a bit to reflect the Debian change once that is in. But I wanted to get feedback from you on this approach.

File: pkg/cloud/vsphere/provisioner/govmomi/utils.go [hunk 1]
```diff
@@ -165,7 +165,7 @@ func (pv *Provisioner) HandleClusterError(cluster *clusterv1.Cluster, err *apier
 	return err
 }
 
-func (pv *Provisioner) GetSSHPublicKey(cluster *clusterv1.Cluster) (string, error) {
+func (pv *Provisioner) GetSSHInternalKey(cluster *clusterv1.Cluster) (string, error) {
```
Comment (ntfrnzn) [inline pkg/cloud/vsphere/provisioner/govmomi/utils.go]: Needs comment indicating that this is the cluster-api master key.  "Internal" is confusing.

Comment (sverrirsig) [reply pkg/cloud/vsphere/provisioner/govmomi/utils.go]: GetClusterAPIControllerSSHKey?

**Comment by ntfrnzn** [comment]
Makes sense to me. I'm not 100% about putting the keys in the cluster provider CRD, but it seems like a decent idea, and I don't have a better one.

File: pkg/cloud/vsphere/provisioner/govmomi/utils.go (hunk 1 above)
Comment (ntfrnzn) [reply pkg/cloud/vsphere/provisioner/govmomi/utils.go]: 👍 Verbose but accurate.

Comment (sverrirsig) [reply pkg/cloud/vsphere/provisioner/govmomi/utils.go]: Done

**Pull request merged**